doc/htmldoc/model_details/aeif_models_implementation.ipynb

```diff
@@ -48,12 +48,12 @@
     "\n",
     "The reference solution is implemented using the LSODAR solver which is described and compared in the following references:\n",
     "\n",
-    "* http://www.radford.edu/~thompson/RP/eventlocation.pdf (papers citing this one)\n",
-    "* http://www.sciencedirect.com/science/article/pii/S0377042712000684\n",
+    "* http://www.edu/~thompson/RP/eventlocation.pdf (papers citing this one)\n",
+    "* https://doi.org/10.1016/j.cam.2012.02.011\n",
     "* http://www.radford.edu/~thompson/RP/rootfinding.pdf\n",
     "* https://computation.llnl.gov/casc/nsde/pubs/u88007.pdf\n",
     "* http://www.cs.ucsb.edu/~cse/Files/SCE000136.pdf\n",
-    "* http://www.sciencedirect.com/science/article/pii/0377042789903348\n",
+    "* https://doi.org/10.1016/0377-0427(89)90334-8\n",
     "* http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.455.2976&rep=rep1&type=pdf\n",
     "* https://theses.lib.vt.edu/theses/available/etd-12092002-105032/unrestricted/etd.pdf"
```

Comment:
I should have followed proper scientific referencing standards ... Here are the references I could find in proper shape:

* Shampine, L. F., & Thompson, S. (2000). Event location for ordinary differential equations. Computers & Mathematics with Applications, 39(5–6), 43–54. https://doi.org/10.1016/S0898-1221(00)00045-6
* Thompson, S. (1987). A collection of test problems for ordinary differential equation solvers which have provisions for rootfinding (ORNL/TM-9912, 6111421; p. ORNL/TM-9912, 6111421). https://doi.org/10.2172/6111421
* Dieci, L., & Lopez, L. (2012). A survey of numerical methods for IVPs of ODEs with discontinuous right-hand side. Journal of Computational and Applied Mathematics, 236(16), 3967–3991. https://doi.org/10.1016/j.cam.2012.02.011
* Hindmarsh, A. C. (1983). ODEPACK, a systematized collection of ODE solvers. In R. S. Stepleman & others (Eds.), Scientific computing (Vol. 1, pp. 55–64). North-Holland. https://computing.llnl.gov/sites/default/files/u88007.pdf
* Petzold, L. (1983). Automatic Selection of Methods for Solving Stiff and Nonstiff Systems of Ordinary Differential Equations. SIAM Journal on Scientific and Statistical Computing, 4(1), 136–148. https://doi.org/10.1137/0904010
* Kahaner, D. K., Lawkins, W. F., & Thompson, S. (1989). On the use of rootfinding ODE software for the solution of a common problem in nonlinear dynamical systems. Journal of Computational and Applied Mathematics, 28, 219–230. https://doi.org/10.1016/0377-0427(89)90334-8

Reply:
added updated references, removed old ones